README.md

```diff
@@ -1,32 +1,35 @@
-![banner](https://user-images.githubusercontent.com/7728097/57463977-2263fc80-727c-11e9-833d-669d816fb7fb.jpg)
```

Comment:
I kinda want to keep this image there as well. Can you try to keep both?

Reply:
Sorry… just added back :)